tests/ContextTest.py

```diff
@@ -1,5 +1,6 @@
+import rdflib
 from rdflib.term import URIRef
-from rdflib.graph import ConjunctiveGraph
+from unittest.mock import Mock
```

Comment:
this should say something like:

    try:
        from unittest.mock import Mock
    except:
        from mock import Mock

Alternatively, you can import `six` and branch off of `six.PY2`.

Reply:
ah okok

Reply:
For make thing in the same way that rest of code:
try:
    from unittest.mock import MagicMock
except ImportError:
    from mock import MagicMock
Found in IdentifierMixinTest.py